keyboards/space_space/rev2/keymaps/default/rules.mk

```diff
@@ -0,0 +1 @@
+COMBO_ENABLE = yes
```

Comment:
For the default keymap, combos shouldn't be enabled.

Reply:
commented out.
could have changed it to 'no' i guess, but i just thought of that 😄 
thanks for taking the time!